packages/admin-ui/src/pages/wifi/components/WifiStatus.tsx

```diff
@@ -0,0 +1,150 @@
+// React
+import React, { useEffect } from "react";
+import { useSelector } from "react-redux";
+import { NavLink } from "react-router-dom";
+// Own components
+import Card from "components/Card";
+import ErrorView from "components/ErrorView";
+import Loading from "components/Loading";
+import Switch from "components/Switch";
+import { confirm } from "components/ConfirmDialog";
+import { withToast } from "components/toast/Toast";
+// API
+import { api, useApi } from "api";
+import { continueIfCalleDisconnected } from "api/utils";
+import { getDappnodeIdentityClean } from "services/dappnodeStatus/selectors";
+// Types
+import { ContainerState, WifiReport } from "types";
+// css
+import "./wifi.scss";
+import { StateBadgeContainer } from "pages/packages/components/StateBadge";
+import { MdWifi } from "react-icons/md";
+// Utils
+import { prettyDnpName } from "utils/format";
+
+const wifiDnpName = "wifi.dnp.dappnode.eth";
```

Comment:
Don't add any important constants in deep files. All constants like this should go into the UI's params file